fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -560,6 +573,11 @@ Future<Optional<std::string>> createMetacluster(Reference<DB> db, ClusterName na
 			}
 
 			wait(metaclusterEmptinessCheck);
+			TenantMode tenantMode = wait(tenantModeFuture);
+			if (tenantMode != TenantMode::DISABLED) {
+				return fmt::format("cluster is configured with tenant mode: `{}' when tenants should be disabled",
```

Comment:
Nit: I think this reads better without the colon since the sentence continues after the value being printed

```suggestion
				return fmt::format("cluster is configured with tenant mode `{}' when tenants should be disabled",
```